CMakeLists.txt

```diff
@@ -394,3 +394,5 @@ if(EMSCRIPTEN)
   set_property(TARGET binaryen_js PROPERTY CXX_STANDARD_REQUIRED ON)
   install(TARGETS binaryen_js DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
+
+configure_file(scripts/binaryen-lit.in ${CMAKE_INSTALL_BINDIR}/binaryen-lit @ONLY)
```

Comment:
What happens if you haven't set an install dir? Would it make more sense to generate this in the normal build bin dir instead?